CBI-261 | Cloud user security page documentation (#2651)
qdrant/landing_page: qdrant-landing/content/documentation/cloud-user-profile.md

> **Note:** This page is for invitations *you* have received. To invite other users to an account you manage, use the **Access Management** page instead. See [Inviting Users to an Account](/documentation/cloud-rbac/user-management/#inviting-users-to-an-account).

## Security

> The **Security** page is where you protect your login with multi-factor authentication (MFA). Like the rest of your profile, MFA is a personal setting — it applies to your user wherever you sign in, across every account you belong to.

Reviewer: ```suggestion
The **Security** page is where you protect your login with multi-factor authentication (MFA). Like the rest of your profile, MFA is a personal setting — it applies to your user whenever you sign in.
```
The reference to accounts could be confusing here.

Enabling MFA is your choice: until you set up a method, nothing changes about how you log in. Once a method is enabled, you are asked to complete it on every subsequent login. Two methods are available, and you can enable both:

* **Authenticator App** — a one-time code from an authenticator app of your choice.
* **Passkey** — a fingerprint, face scan, screen lock, or hardware security key.

> Setting up either method sends you back through the login screen, because that is where your app or device is registered. When it completes, you land back on the **Security** page.

Reviewer: ```suggestion
You will be asked to authenticate again in order to set up a MFA method. After completion, you land back on the **Security** page.
```

![User Security Page](/documentation/cloud/user-security-page.png)

### Authenticator App

> Select **Setup Authenticator App** to get started. You are taken back through login, where you scan a QR code with your authenticator app and confirm the code it generates.

Reviewer: ```suggestion
Select **Setup Authenticator App** to get started. You will be presented with a QR code that you need to scan with your authenticator app and confirm the code it generates.
```

A **recovery code** always accompanies an authenticator app, and is issued in the same step. It is shown once, so store it somewhere safe — it is your way back in if you lose access to your authenticator app.

Once set up, the method is marked **Enabled** and two actions become available:

> * **Regenerate recovery code** — disables your existing recovery code and sends you back through login to issue a new one.

Reviewer: ```suggestion
* **Regenerate recovery code** — disables your existing recovery code and generate a new one.
```

* **Disable** — removes the authenticator app across all devices, along with its recovery code. You can no longer use it for MFA.

### Passkeys

> A passkey lets you approve a login with your device or a security key instead of typing in a code. Select **Create a New Passkey** and choose one of:

Reviewer: ```suggestion
A passkey lets you approve a login with your device or a security key. Select **Create a New Passkey** and choose one of:
```
